Explorer/src/app/pods/page.tsx

```diff
@@ -0,0 +1,14 @@
+"use client";
```

Comment:
I think non of these should be client components.
Especially once we start doing actual data fetching in here